airflow/providers/google/cloud/hooks/vertex_ai/hyperparameter_tuning_job.py

```diff
@@ -413,3 +428,106 @@ def delete_hyperparameter_tuning_job(
             metadata=metadata,
         )
         return result
+
+
+class HyperparameterTuningJobAsyncHook(GoogleBaseHook):
+    """Async hook for Google Cloud Vertex AI Hyperparameter Tuning Job APIs."""
+
+    def __init__(
+        self,
+        gcp_conn_id: str = "google_cloud_default",
+        impersonation_chain: str | Sequence[str] | None = None,
+        **kwargs,
+    ):
+        super().__init__(
+            gcp_conn_id=gcp_conn_id,
+            impersonation_chain=impersonation_chain,
+            **kwargs,
+        )
+        self._client: JobServiceAsyncClient | None = None
+
+    def get_job_service_client(self, region: str | None = None) -> JobServiceAsyncClient:
```

Comment:
Should we try something like https://github.com/apache/airflow/blob/c7ade012cb81158e2bc11b36febcfae82bc759ab/airflow/providers/amazon/aws/operators/batch.py#L224-L231

Reply:
Great idea, thanks!